format/spec.md

```diff
@@ -51,6 +51,7 @@ Version 3 of the Iceberg spec extends data types and existing metadata structure
 * New data types: nanosecond timestamp(tz)
 * Default value support for columns
 * Multi-argument transforms for partitioning and sorting
+* Binary deletion vectors
```

Comment:
The Puffin spec says `delete vector`, we say `deletion vector` here. Which form is more accurate?

Reply:
`deletion vector` seems to be slightly more accurate imo, so maybe we should also update https://github.com/apache/iceberg/pull/11238 to call it `deletion-vector-v1`

Reply:
Aren't they synonyms? I can change it, I guess.

Reply:
Updated.